tests/scripts/multus/230-stretch-pub-only-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
What is the significance of the number in the filename?

Reply:
Trying to have the ordering of the test files match up to the order in which they should be run in CI.

Reply:
I see, a convenience during development. What about a naming convention like `test-<id>-<name>.sh`, so this script would be `test-230-stretch-pub-only.sh`? Then maybe it's self explanatory that it's just a simple test number.